manifests/config.pp

```diff
@@ -63,15 +63,17 @@
   contain foreman_proxy::module::httpboot
 
   contain foreman_proxy::module::puppet
+  foreman_proxy::provider { 'puppet_proxy_puppet_api':
+  }
   foreman_proxy::provider { [
+      'puppet_proxy_legacy',
+      'puppet_proxy_puppetrun',
       'puppet_proxy_customrun',
       'puppet_proxy_mcollective',
       'puppet_proxy_puppet_api',
```

Comment:
I think this one shouldn't be in the `ensure => 'absent'` group since it's still used

Reply:
I think this one should be in the `ensure => 'absent'` because they are in current implentation too (see https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/config.pp#L74).

Should I move it then?

Reply:
specifically `puppet_proxy_puppet_api` is the only remaining provider that is still supported. it isn't in the current list of `ensure => 'absent'` which makes sure these providers aren't installed on the proxy. It should be removed from this list.